soccer/gameplay/tactics/advance_zone_midfielder.py

```diff
@@ -12,7 +12,7 @@
 import evaluation.passing
 import evaluation.shooting
 import functools
-import plays.offense.adaptive_formation
+import plays.Legacy.adaptive_formation
```

Comment:
Had to change an import due to the move of adaptive formation. This should probably be looked at to make sure it doesn't break anything. It feels a bit circular/bad that this requires the play that uses it?

Reply:
The reason why we did this is that it needs to pull some data from it. I can modify it, but I may need to re-think how I want to do it to make it general.

Reply:
We should definitely try to transition it to only a 1 way data flow, from parent to child play. Otherwise you can get some real funk down the rode when we change stuff